2-mandatory/1-weather-report.js

```diff
@@ -11,9 +11,26 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
-}
+
+
+
+ function getTemperatureReport(cities) {
+
+let news = []
+
+for (let i = 0 ; i < cities.length; i++ ){
+
+let result  = temperatureService(cities[i])
+
+news.push (`The temperature in ${cities[i]} is ${result} degrees`)
```

Comment:
this is all really good! I would just watch the formatting - make sure there is no space between `push` and the bracket, make sure all lines in the function are indented/tabbed a line, make sure the return is on a new line!

I'm not sure what text editor you use but if it is vscode then try looking at something like https://www.alphr.com/auto-format-vs-code/#:~:text=Click%20the%20%E2%80%9CSettings%E2%80%9D%20gear%20icon,%3A%20Format%20On%20Save%E2%80%9D%20option. :)

Reply:
Oh that's great advice thank you very much